### DISCUSSION TITLE
Форма

### DISCUSSION THREAD

**Discussion by webdesignermsk** [leon-mbs/zstore#277] [Ideas]
Привет. Подскажите, как организовать форму, в которой можно заказать звонок, либо задать вопрос, быть может что то типо сбора заявок.

**Comment by webdesignermsk** [comment]
Ещё, было бы классно к атрибутам прикреплять картинку svg, либо цвет. А так же произвольные поля к товару...

**Comment by leon-mbs** [comment]
для какой  страницы  эта  форма?

большинство  пользователей  не  атишники  а  простой  бизнес не  будут  они  пользовать никакие svg  они даже не  згнают что  это

по  поводу  полей - ну добавили  и что  дальше с  ними  делать  кроме  показывать?
там  есть  поле  коментария  доп  информацию можно  вписать  туда

опять   же больльшинство  пользаков  не  айтишники  не будут они добавлять  никакие  поля

и вообще  идеология проекта  по  максимуму  простая  архитектура  для  доработки  и сопровождения - настрока  полецй в  таблицах  - неоправданое  усложнение

если  вы  об  онлайн  каталоге  то там  можно  добавлять  атриьуты
